### DISCUSSION TITLE
Needs more YouTube videos demonstrating this API

### DISCUSSION THREAD

**Discussion by iMacker2020** [MethanePowered/MethaneKit#93] [Ideas]
Hi I think this project would benefit from having YouTube videos that show how well this API functions. A video tutorial series would also be helpful for new users of this API.

Thank you.

**Comment by egorodet** [comment]
Hello @iMacker2020, this is a code project created for other developers. The best way to evaluate the library is to try it yourself: clone repository, build it and run tutorials and samples on your own machine. Some tutorials have brief code explanation in README files. In case if someone is too lazy to clone and build, this project has pre-built executable binaries attached to [releases](https://github.com/egorodet/MethaneKit/releases). Also the code can be built for experimental purposes under [Cloud IDE GitPod](https://gitpod.io/#https://github.com/egorodet/MethaneKit) right in your browser. I think that YouTube videos are nice to have in general, but would be excessive for this code project for now. Anyway thanks for the idea, maybe I will consider creating them later.
